shared/metrics/src/main/java/io/pravega/shared/metrics/StatsProvider.java

```diff
@@ -19,6 +19,12 @@
      */
     void start();
 
+    /**
+     * Initialize the stats provider without exporting functionality.
+     * In other words, starts with Micrometer SimpleMeterRegistry only.
```

Comment:
Can you please combine these two sentences into a single one that clearly explains what this is doing. `In other words` really doesn't belong in a Javadoc comment.

Reply:
Reworded